services/reading-contest-api/domain/contest.go

```diff
@@ -43,3 +43,14 @@ type ContestID uint64
 func (id ContestID) IsGlobal() bool {
 	return id == 0
 }
+
+type ContestStats struct {
+	ByLanguage   []ContestLanguageStat `json:"byLanguage"`
+	Participants int                   `json:"participants"`
+	TotalPages   float64               `json:"totalPages"`
```

Comment:
Can we call this `TotalAmount`? `Amount` is what we've been using in the code base to refer to a user's score/pages.